resource.address/spi/src/main/java/org/kaazing/gateway/resource/address/Comparators.java

```diff
@@ -24,6 +24,8 @@
 import java.util.Comparator;
 import java.util.List;
 
+import org.kaazing.gateway.resource.address.uri.URIUtils;
```

Comment:
Hello, Jitu, there are some static URIUtils methods used in this class. Therefore the import. The change in the file occurs due to a package refactoring.